facefusion/processors/modules/lip_syncer.py

```diff
@@ -183,6 +226,20 @@ def forward(temp_audio_frame : AudioFrame, close_vision_frame : VisionFrame) ->
 	return close_vision_frame
 
 
+def forward_edtalk(temp_audio_frame : AudioFrame, crop_vision_frame : VisionFrame, lip_syncer_weight : float) -> VisionFrame:
+	lip_syncer = get_inference_pool().get('lip_syncer')
+
+	with conditional_thread_semaphore():
+		crop_vision_frame = lip_syncer.run(None,
+		{
+			'source': temp_audio_frame,
+			'target': crop_vision_frame,
+			'weight': [ numpy.float32(lip_syncer_weight) ]
```

Comment:
doing operations within an list is ugly ... follow how we are doing this in the face enhancer processor